### PR TITLE
Fixes #16790: Our agent package contains CFEngine systemd units

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16790

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -649,6 +649,8 @@ install-cfengine: build-cfengine $(INSTALL_DEPS) rudder.8.gz
 		LD_LIBRARY_PATH="$(DESTDIR)/opt/rudder/lib" ./$${binary} -M | gzip > $(DESTDIR)/opt/rudder/share/man/man8/$${binary}.8.gz; \
 	done
 	cd $(DESTDIR)/opt/rudder && rm -rf share/doc/cfengine
+	# Remove CFEngine systemd units (ours are in /lib/systemd)
+	cd $(DESTDIR) && rm -rf usr/lib/systemd/system/
```
Comment (peckpeck) [inline rudder-agent/SOURCES/Makefile]: what if other systemd units have be installed in DESTDIR before ?
can't you remove files explicitly ?

Comment (amousset) [reply rudder-agent/SOURCES/Makefile]: We don't use the same dir for systemd units, but we can name them indeed

Comment (peckpeck) [reply rudder-agent/SOURCES/Makefile]: but other products in DESTDIR may do

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
@peck updated

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**